[2021-11-29 23:50:36,381] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-11-27T09:00:00+00:00 [queued]>
[2021-11-29 23:50:36,401] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-11-27T09:00:00+00:00 [queued]>
[2021-11-29 23:50:36,401] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 23:50:36,401] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2021-11-29 23:50:36,401] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 23:50:36,415] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2021-11-27T09:00:00+00:00
[2021-11-29 23:50:36,416] {standard_task_runner.py:54} INFO - Started process 4602 to run task
[2021-11-29 23:50:36,548] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2021-11-27T09:00:00+00:00', '--job_id', '27', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmp19d_dgja']
[2021-11-29 23:50:36,548] {standard_task_runner.py:78} INFO - Job 27: Subtask twitter_aluraonline
[2021-11-29 23:50:36,608] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2021-11-27T09:00:00+00:00 [running]> on host kauai-VirtualBox
[2021-11-29 23:50:36,715] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2021-11-29 23:50:36,721] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-27T09:00:00.00Z&end_time=2021-11-28T09:00:00.00Z
[2021-11-29 23:50:37,096] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-29 23:50:37,482] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-27T09:00:00.00Z&end_time=2021-11-28T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy797bb3h162jo74htqnbcuu53st
[2021-11-29 23:50:37,482] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-29 23:50:37,879] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-27T09:00:00.00Z&end_time=2021-11-28T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy797aeh4ghawfxo1gpxhn8wzhj1
[2021-11-29 23:50:37,880] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-29 23:50:38,284] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-27T09:00:00.00Z&end_time=2021-11-28T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy7957gzbplaztppx6uelipu6z5p
[2021-11-29 23:50:38,285] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-29 23:50:38,699] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-27T09:00:00.00Z&end_time=2021-11-28T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy7956km1de954w56l2blwm90akd
[2021-11-29 23:50:38,701] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-29 23:50:39,079] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-11-27T09:00:00.00Z&end_time=2021-11-28T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy7933ckv2lcl0eiyvjop0xxfiil
[2021-11-29 23:50:39,080] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-11-29 23:50:39,446] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20211127T090000, start_date=20211130T025036, end_date=20211130T025039
[2021-11-29 23:50:41,367] {local_task_job.py:102} INFO - Task exited with return code 0
